### PR TITLE
e2e testcase: Clearing field during 'Notify with edits'

### PR DESCRIPTION
**Pull request by cibelius**
## Description

core PR: https://github.com/opencrvs/opencrvs-core/pull/13031

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/birth/declarations/clear-field-after-notify-edit.spec.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  await test.step("Fill child's name and date of birth", async () => {
+    await page.locator('#firstname').fill(childName.firstNames)
+    await page.locator('#surname').fill(childName.familyName)
+    await fillDate(page, childDob)
+  })
+
+  await test.step('Continue to review and Notify', async () => {
+    await goToSection(page, 'review')
+    await triggerDeclarationAction(page, 'Notify')
+  })
```
Comment (Zangetsu101) [inline e2e/testcases/birth/declarations/clear-field-after-notify-edit.spec.ts:48]: Ideally we would create the record via the api

**Review by Zangetsu101 — APPROVED** [review]

File: e2e/testcases/birth/declarations/clear-field-after-notify-edit.spec.ts (hunk 1 above)
Comment (cibelius) [reply e2e/testcases/birth/declarations/clear-field-after-notify-edit.spec.ts:48]: Decided to keep it as is, since the initial notifies are simple

But will keep in mind moving on!
